backend/src/main/java/com/cs471/studentLoanSystem/login/LoginSystem.java

```diff
@@ -12,14 +25,20 @@
 
 @RestController
 public class LoginSystem {
+    private final String BANK_OFFICER_ROLE = "BANKOFFICER";
+    private final String STUDENT_ROLE = "STUDENT";
+    private final String REGISTRAR_ROLE = "REGISTRAR";
+    private final String LOAN_OFFICER_ROLE = "LOANOFFICER";
```

Comment:
Two things on this:

- Aren't bank officers and loan officers the same role? They seem to be used interchangeably in the project description
- I think this should be defined somewhere else, like a common definitions helper class (maybe as an enum?) in case we need to use these somewhere else

Reply:
In theory, the only time the backend will need to see these is here. The frontend uses these to get an idea where the user belongs.

Also, Loan officer and bank officer are similar but different. On the stories, there are a few tagged with bank-officer/loan-officer respectively. There are a few things a bank officer can do that a loan officer can do and vice versa.

Reply:
> In theory, the only time the backend will need to see these is here. The frontend uses these to get an idea where the user belongs.
>
See my comment on #4, keeping the backend ignorant of the role of the requesting user makes things complicated

Reply:
Given discussion on #4, I think I am now on the side of making sure the backend has these roles enumerated somewhere.